packages/node-sdk/src/utils.ts

```diff
@@ -14,6 +14,13 @@ export function ok(condition: boolean, message: string): asserts condition {
   }
 }
 
+export function assertId(id: string | number, entity: string) {
```

Comment:
Maybe add an optional param `allowUndefined`. Would remove the need for more ifs in code above

Reply:
I will leave as it. Hate boolean args ;)